src/settings/MappingProfiles/detailsSections/edit/OrderDetailSection/EResourcesDetails.js

```diff
@@ -88,7 +88,7 @@ export const EResourcesDetails = ({
               <FieldOrganization
                 id={accessProviderId}
                 setReferenceTables={setReferenceTables}
-                name={getFieldName(68)}
+                name={getFieldName(64)}
```

Comment:
Would be great to have some constants for those numbers

Reply:
Hi @artem-blazhko I added field maps for indexes, could you review, please?